static/app/components/sentryAppComponentIcon.tsx

```diff
@@ -46,28 +46,27 @@ const getFallbackIcon = (slug: string) => {
   }
 };
 
-const SentryAppIcon = ({sentryAppComponent: {sentryApp}}: Props) => {
-  return (
-    <Feature features={['organizations:sentry-app-logo-upload']}>
-      {({hasFeature}) => {
-        const selectedAvatar = sentryApp?.avatars?.find(({color}) => color === false);
-        const isDefault = !(selectedAvatar && selectedAvatar?.avatarType === 'upload');
-        return hasFeature ? (
-          <AvatarWrapper
-            isDark={ConfigStore.get('theme') === 'dark'}
-            isDefault={isDefault}
-          >
-            <Avatar size={20} sentryApp={sentryApp} isColor={false} />
-          </AvatarWrapper>
-        ) : (
-          getFallbackIcon(sentryApp.slug)
-        );
-      }}
-    </Feature>
-  );
-};
+/**
+ * Icon Renderer for SentryAppComponents with UI
+ * (e.g. Issue Linking, Stacktrace Linking)
+ */
+const SentryAppComponentIcon = ({sentryAppComponent: {sentryApp}}: Props) => (
```

Comment:
@scefali  This change was simply done for clarity. After the suggestion for using `SentryAppIcon`, it occured to me we could just directly use the avatar, so `SentryAppAvatar` instead. I remembered `SentryAppIcon` does in fact exist, but it's not the same as the Avatar, so renamed and added a docstring for clarity.